papers/oliver_beckstein/oliver_beckstein.rst

```diff
@@ -93,6 +93,32 @@ The user interface and modular design work equally well in complex scripted work
 MDAnalysis is available in source form under the GNU General Public License v2 from GitHub https://github.com/MDAnalysis/mdanalysis and PyPi;  conda packages are also available. The documentation is extensive http://docs.mdanalysis.org including an introductory tutorial http://www.mdanalysis.org/MDAnalysisTutorial/ and a very friendly and welcoming user and developer community. 
 
 
+New data Structures
+-------------------
+
+Originally MDAnalysis followed a strict object oriented approach with a separate instance of an Atom object for each particle in the simulation data.
+The AtomGroup then simply stored its contents as a list of these Atom instances.
+With simulation data commonly containing 10\ :sup:`6` particles
+this solution did not scale well and so
+recently this design was overhauled to improve the scalability of MDAnalysis.
```

Comment:
> This solution doesn't scale well with the size of common molecular simulations that contain around 10\ :sup:`6` particles and more. So recently this design was overhauled to improve the scalability of MDAnalysis.

A theme for good writing practices is to start with something old and introduce the new in the end of a sentence. So I just turned your sentence around.